udfs/community/README.md

```diff
@@ -13,24 +13,29 @@ SELECT bqutil.fn.int(1.684)
 
 ## UDFs
 
-* [csv_to_struct](#csv_to_structstrlist-string)
-* [find_in_set](#find_in_setstr-string-strlist-string)
-* [freq_table](#freq_tablearr-any-type)
-* [get_array_value](#get_array_valuek-string-arr-any-type)
-* [get_value](#get_valuek-string-arr-any-type)
-* [int](#intv-any-type)
-* [median](#medianarr-any-type)
-* [nlp_compromise_number](#nlp_compromise_numberstr-string)
-* [nlp_compromise_people](#nlp_compromise_peoplestr-string)
-* [radians](#radiansx-any-type)
-* [random_int](#random_intmin-any-type-max-any-type)
-* [random_value](#random_valuearr-any-type)
-* [translate](#translateexpression-string-characters_to_replace-string-characters_to_substitute-string)
-* [typeof](#typeofinput-any-type)
-* [url_keys](#url_keysquery-string)
-* [url_param](#url_paramquery-string-p-string)
-* [url_parse](#url_parseurlstring-string-parttoextract-string)
-* [zeronorm](#zeronormx-any-type-meanx-float64-stddevx-float64)
+- [Community UDFs](#community-udfs)
```

Comment:
I'm confused at what this restructure is doing.

Reply:
Opps, that'll be my overzealous auto-formatter. Not sure how I missed that when staging my changes - I'll revert it back to the way it was.